android/app/src/main/java/org/fossasia/openevent/utils/DateUtils.java

```diff
@@ -173,4 +173,9 @@ public static String getRelativeTimeFromTimestamp(String timeStamp) throws Parse
                 (timeCreatedDate.getTime()),
                 System.currentTimeMillis(), android.text.format.DateUtils.SECOND_IN_MILLIS);
     }
+
+    public static boolean isUpcomingSession(Date start, Date end, Date current) {
```

Comment:
@iamareebjamal  I think that DateUtils. java shouldn't be restricted just to conversions as it is technically the 'utility' class for dates. What do you think?